pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[tool.poetry]
+name = "pycytominer"
+version = "0.3.0"
+description = "Python package for processing image-based profiling data"
+authors = [
+    "Gregory Way <gregory.way@gmail.com>",
+]
+maintainers = [
+    "Gregory Way <gregory.way@gmail.com>",
+]
```

Comment:
@gwaybio Could you provide the names of the authors and maintainers you would like to include here in the pyproject.toml? Long-term, @d33bs suggested finding a way to auto-populate this from the CITATION.cff, but in the meantime we just want the current, up-to-date list.

Reply:
I've copied over the authors from the CITATION.cff file. We can merge the updated author list in a separate PR. I've created [this issue](https://github.com/cytomining/pycytominer/issues/316) to track completion prior to 1.0 release.